homeassistant/components/history/__init__.py

```diff
@@ -43,7 +43,14 @@
 IGNORE_DOMAINS = ("zone", "scene")
 
 
-def get_significant_states(
+def get_significant_states(hass, *args, **kwargs):
+    """Wrap _get_significant_states with a sql session."""
+    with session_scope(hass=hass) as session:
+        return _get_significant_states(session, hass, *args, **kwargs)
+
+
+def _get_significant_states(
+    session,
```

Comment:
Our standard is to put `hass` first in function signatures if `hass` is present.

Reply:
Adjusted, retested.  Everything still working as expected